src/Database/PG/Query/Connection.hs

```diff
@@ -398,6 +430,7 @@ data PGConn
   = PGConn
   { pgPQConn       :: !PQ.Connection
   , pgAllowPrepare :: !Bool
+  , pgCancel       :: !Bool
```

Comment:
should this be called `cancellable`? could you add a comment please?

Reply:
Yes will definitely document this, it's very much a first draft.

Thinking for `cancel` as opposed to `cancellable` was that this setting says that "if we receive a certain exception, we cancel the command". But yeah, I'm not really clear on how cancelling/interruptability should be expressed in a library API... Ties in to the question below of which exceptions to handle, too.